vortex-array/src/arrays/primitive/compute/take.rs

```diff
@@ -72,11 +73,11 @@ register_kernel!(TakeKernelAdapter(PrimitiveVTable).lift());
 /// # Parameters
 /// * `indices` - Indices to gather values from
 /// * `values` - Source values to index
-/// * `nullability` - Nullability of the resulting array
 ///
 /// # Returns
 /// A `PrimitiveArray` containing the gathered values where each index has been replaced with
 /// the corresponding value from the source array.
+#[multiversion(targets("x86_64+avx+avx2"))]
```

Comment:
This is one version besides the fallback right? Would make sense to add
`x86_64+avx2` and `x86_64+avx" as 2 targets? avx2 implies avx I think.